seldon-core-operator/rockcraft.yaml

```diff
@@ -107,7 +109,6 @@ parts:
       useradd -R $CRAFT_OVERLAY -M -r -u 1001 -g ubuntu ubuntu
     override-prime: |
       craftctl default
-      chown -R 1001:1001 opt/seldon-core-operator
```

Comment:
Isn't this part of the security requirement?

Reply:
That directory does not exist in final rock, so this was removed, because it had no effect.